base_tier_validation/models/tier_validation.py

```diff
@@ -195,7 +195,14 @@ def _calc_reviews_rejected(self, reviews):
 
     def _compute_need_validation(self):
         for rec in self:
-            tiers = self.env["tier.definition"].search([("model", "=", self._name)])
+            tiers = self.env["tier.definition"].search(
+                [
+                    ("model", "=", self._name),
+                    "|",
+                    ("company_id", "=", self.env.company.id),
+                    ("company_id", "=", False),
```

Comment:
```suggestion
                    ("company_id", "in", [False] + self.env.company.ids),
```

And the same for the other.